### PR TITLE
Updated to fill out the query parameters prior to creating Query object

### PR DESCRIPTION
**Pull request by ivakegg**

### CODE REVIEW

File: web-services/query/src/main/java/datawave/webservice/query/util/LookupUUIDUtil.java [hunk 1]
```diff
@@ -428,12 +429,25 @@ public <T> T createUUIDQueryAndNext(final AbstractUUIDLookupCriteria unvalidated
...
+                for (QueryImpl.Parameter pm : QueryUtil.parseParameters(params)) {
+                    parms.putSingle(pm.getParameterName(), pm.getParameterValue());
+                }
+            }
+            parms.putAll(queryParameters);
+            log.debug("Final query parameters: " + parms);
+            query.setOptionalQueryParameters(parms);
+            for (String key : parms.keySet()) {
+                if (parms.get(key).size() == 1) {
+                    query.addParameter(key, parms.getFirst(key));
```
Comment (billoley) [inline web-services/query/src/main/java/datawave/webservice/query/util/LookupUUIDUtil.java]:
```suggestion
            MultivaluedMap<String,String> optionalQueryParameters = new MultivaluedMapImpl<>();
            if (defaultOptionalParams != null) {
                optionalQueryParameters.putAll(defaultOptionalParams);
            }
            String delimitedParams = queryParameters.getFirst(QueryParameters.QUERY_PARAMS);
            if (delimitedParams != null) {
                for (QueryImpl.Parameter pm : QueryUtil.parseParameters(delimitedParams)) {
                    optionalQueryParameters.putSingle(pm.getParameterName(), pm.getParameterValue());
                }
            }
            optionalQueryParameters.putAll(queryParameters);
            log.debug("Final query parameters: " + optionalQueryParameters);
            query.setOptionalQueryParameters(optionalQueryParameters);
            for (String key : optionalQueryParameters.keySet()) {
                if (optionalQueryParameters.get(key).size() == 1) {
                    query.addParameter(key, optionalQueryParameters.getFirst(key));
```

**Review by billoley — CHANGES_REQUESTED** [review]
Request better variable names than params vs parms vs queryParameters

**Review by apmoriarty — DISMISSED** [review]
LGTM

**Review by keith-ratcliffe — DISMISSED** [review]
Looks good

**Review by keith-ratcliffe — APPROVED** [review]

**Review by apmoriarty — APPROVED** [review]

**Pull request merged**